tests/components/bond/common.py

```diff
@@ -60,70 +63,46 @@ async def setup_platform(
 
 
 def patch_bond_device_ids(return_value=None):
-    """Patch Bond API getDeviceIds command."""
+    """Patch Bond API devices command."""
     if return_value is None:
         return_value = []
 
     return patch(
-        "homeassistant.components.bond.Bond.getDeviceIds", return_value=return_value,
+        "homeassistant.components.bond.Bond.devices", return_value=return_value,
     )
 
 
-def patch_bond_turn_on():
-    """Patch Bond API turnOn command."""
-    return patch("homeassistant.components.bond.Bond.turnOn")
-
-
-def patch_bond_turn_off():
-    """Patch Bond API turnOff command."""
-    return patch("homeassistant.components.bond.Bond.turnOff")
-
-
-def patch_bond_set_speed():
-    """Patch Bond API setSpeed command."""
-    return patch("homeassistant.components.bond.Bond.setSpeed")
-
-
-def patch_bond_set_flame():
-    """Patch Bond API setFlame command."""
-    return patch("homeassistant.components.bond.Bond.setFlame")
-
-
-def patch_bond_open():
-    """Patch Bond API open command."""
-    return patch("homeassistant.components.bond.Bond.open")
-
-
-def patch_bond_close():
-    """Patch Bond API close command."""
-    return patch("homeassistant.components.bond.Bond.close")
-
+def patch_bond_action():
+    """Patch Bond API action command."""
+    return patch("homeassistant.components.bond.Bond.action")
 
-def patch_bond_hold():
-    """Patch Bond API hold command."""
-    return patch("homeassistant.components.bond.Bond.hold")
-
-
-def patch_bond_set_direction():
-    """Patch Bond API setDirection command."""
-    return patch("homeassistant.components.bond.Bond.setDirection")
 
+def patch_bond_device_state(return_value=None, side_effect=None):
+    """Patch Bond API device state endpoint."""
+    if return_value is None:
+        return_value = {}
 
-def patch_turn_light_on():
-    """Patch Bond API turnLightOn command."""
-    return patch("homeassistant.components.bond.Bond.turnLightOn")
+    return patch(
+        "homeassistant.components.bond.Bond.device_state",
+        return_value=return_value,
+        side_effect=side_effect,
+    )
 
 
-def patch_turn_light_off():
-    """Patch Bond API turnLightOff command."""
-    return patch("homeassistant.components.bond.Bond.turnLightOff")
+async def help_test_entity_available(
+    hass: core.HomeAssistant, domain: str, device: Dict[str, Any], entity_id: str
+):
+    """Run common test to verify available property."""
+    await setup_platform(hass, domain, device)
 
+    assert hass.states.get(entity_id).state != STATE_UNAVAILABLE
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)